Example/Tests/NetworkTests/ManagementApiTests.swift

```diff
@@ -85,6 +87,109 @@ class ManagementApiTests: NetworkBaseTest {
         XCTAssertNil(error, "error should be nil")
         XCTAssertNotNil(result, "response should not be nil")
     }
+
+    func testRenameShouldReturnContext() throws {
+
+        try XCTSkipUnless(allowFolderDecouplingCalls, "prevents redundant call to Cloudinary PAID Folder Decoupling service. to allow Folder Decoupling service testing - set to true")
```

Comment:
@adimiz1 @const-cloudinary Let me know what the service I am trying to use thats blocking me because of my free account type, il change that afterward.